### PR TITLE
Resposive design

### PR DESCRIPTION
**Pull request by legion91**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,16 +29,34 @@ <h1 class="hero__block-heading">There is no other platforms for you as like</h1>
...
-      <section>
-
+      <section class="awesone-features">
+          <h4 class="awesone-features__sub-heading sub-heading">NEW FEATURES</h4>
+          <h2 class="awesone-features__heading heading">Some awesone features</h2>
+          <div class="awesone-features__column">
+              <div class="awesone-features__card">
+                  <img class="awesone-features__cardbg" src="build/img/photos.png" alt="photos">
+                  <h5 class="awesone-features__card-title">Some awesone features</h5>
+                  <p class="description">Donec sed odio dui. Aenean lacinia bibendum nulla sed consectetur. Nulla vitae elit libero, a pharetra augue.</p>
```
Comment (MaxMishchenko) [inline index.html]: Rename to awesone-features__description

File: src/css/awesone-features.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    margin: 0 0 45px;
+  }
+}
+
+.awesone-features__column {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: baseline;
+  Flex-wrap: wrap;
```
Comment (MaxMishchenko) [inline src/css/awesone-features.scss]: Flex-wrap: wrap; rewrite for beauty to flex-wrap: wrap;

File: index.html (hunk 1 above)
Comment (legion91) [reply index.html]: This class is named "description" bacause he add to global.scss.

File: src/css/awesone-features.scss (hunk 2 above)
Comment (legion91) [reply src/css/awesone-features.scss]: What? Flex-wrap: wrap to Flex-wrap: wrap?

**Pull request merged**